enigma.cpp

```diff
@@ -140,31 +139,31 @@ void _encrypt() {
 		userin = 0;
 		cout << "-- Encrypting --" << endl;
 		cout.precision(2);
-		
+
 		for (int i = 0; i < message.length(); i++) {
 			messageArray[i] = message[i];
 			//_replace(); <-- This part below could be put into another function called replace but that's an optimization that's not necessary at this point. 
-				// Tried bypassing issue by using decimals. Formatting looks gross. 
+			// Tried bypassing issue by using decimals. Formatting looks gross. 
 			if (messageArray[i] == 'a' || messageArray[i] == 'A') {
-				messageOut[i] = .01;
+				messageOut[i] = .41;
```

Comment:
I changed the 01, 02, 03,... entries with 41, 42, 43... respectively since the zero character was not being printed to standard out and we need that for decryption to work.